src/utils/fetch-graphql.js

```diff
@@ -114,9 +126,15 @@ const handleGraphQLErrors = async ({
       error.message === errorMap.from
 
     if (errorWasMapped && panicOnError) {
-      reporter.panic(formatLogMessage(errorMap.to))
+      structuredReporter.panic(
+        formatLogMessage(errorMap.to),
+        CODES.GQLConfiguration
```

Comment:
Rathe than configuration, these are actual GraphQL errors being returned from the remote schema. Maybe we need another code for remoteGraphQLError?